package.json

```diff
@@ -86,14 +86,14 @@
     "@typescript-eslint/eslint-plugin": "5.29.0",
     "@typescript-eslint/parser": "5.29.0",
     "app-module-path": "2.2.0",
-    "catw": "1.0.1",
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     "chance": "1.0.16",
+    "chokidar-cli": "^3.0.0",
     "esbuild-loader": "2.19.0",
     "eslint": "8.18.0",
     "http-proxy": "1.18.1",
-    "i18next-scanner": "4.1.0",
+    "i18next-scanner": "^4.4.0",
```

Comment:
Can you use explicit versions numbers (here without `^`). Sometimes it is hard to track down the bug due to mismatch in local and prod version of some dependency.

Reply:
Done!